contracts/interchain-token-service/src/interface.rs

```diff
@@ -34,12 +34,37 @@ pub trait InterchainTokenServiceInterface: AxelarExecutableInterface {
 
     fn token_manager_type(env: &Env, token_id: BytesN<32>) -> TokenManagerType;
 
+    /// Retrieves the flow limit for the token associated with the specified token ID.
+    /// Returns `None` if no limit is set.
     fn flow_limit(env: &Env, token_id: BytesN<32>) -> Option<i128>;
 
+    /// Retrieves the amount that has flowed out of the contract during the current epoch
```

Comment:
```suggestion
    /// Retrieves the amount that has flowed out of the chain to other chains during the current epoch
```
make it clear it's for outgoing interchain transfers